project_and_code_guidelines.md

```diff
@@ -593,18 +577,22 @@ As a general rule you should try to group similar attributes together. A good wa
 
 ### 2.4.1 Unit tests 
 
-The test classes should match the name of the class that the tests are targeting followed by `Test`. For example, If we create a test class that contains test for the `DataManager`, we should name it `DataManagerTest`.
+Test classes should match the name of the class that the tests are targeting followed by `Test`. For example, If we create a test class that contains test for the `DatabaseHelper`, we should name it `DatabaseHelperTest`.
```

Comment:
- Test classes should match the name of the class that the tests are targeting, followed by `Test`. (add a comma before follows)
- For example, if we create a test class that contains test (lower case i and s after test)